CONTRIBUTING.md.txt

```diff
@@ -0,0 +1,42 @@
+# Contributing to Checkstyle Test Configurations Generator
```

Comment:
Should be CONTRIBUTING.md
And under .github folder.

Reply:
done

Reply:
Not done

Reply:
Sorry, i missed it. i guess done now.